main.go

```diff
@@ -31,8 +31,8 @@ func main() {
 	fmt.Println()
 
 	// Remove SSHRsaPrivateKey from envs
-	if err := os.Unsetenv("ssh_rsa_private_key"); err != nil {
-		failf("Failed to remove ssh_rsa_private_key")
+	if err := unsetSSHEnvs(string(cfg.SSHRsaPrivateKey)); err != nil {
+		failf("Failed to remove SSHRsaPrivateKey from envs - error: %s", err)
```

Comment:
wdyt about `failf("Failed to remove private key data from envs, error: %s", err)` ?